rootfs/api/urls.py

```diff
@@ -29,22 +29,6 @@
         views.ReleaseViewSet.as_view({'post': 'rollback'})),
     url(r"^apps/(?P<id>{})/releases/?".format(settings.APP_URL_REGEX),
         views.ReleaseViewSet.as_view({'get': 'list'})),
-    # restart containers
```

Comment:
This change to the REST API implies an update to [the docs](https://github.com/deis/workflow/blob/master/src/reference-guide/controller-api/v2.0.md). I'll make an issue to track that: deis/workflow#93.